### ISSUE TITLE
Package as electron app

### ISSUE THREAD

**Issue by ericsoco** [stamen/living-liquid#1]
Some info on building an Electron app with Browserify [here](http://maxgfeller.com/blog/2016/08/30/electron-browserify/), including a link to [`electron-packager`](https://npm.im/electron-packager). Note also that we may need to set [`nodeIntegration: false`](http://electron.atom.io/docs/faq/#i-can-not-use-jqueryrequirejsmeteorangularjs-in-electron).

**Comment by K-Dasu** [comment]
## Screen Shot of Browserify/Electron issue:
<img width="1306" alt="screen shot 2017-01-03 at 3 16 28 pm" src="https://cloud.githubusercontent.com/assets/3989930/21627171/de6f9664-d1ca-11e6-8d7b-40d6e28a88ff.png">

reference to similar [issue](https://github.com/electron/electron/issues/7626)
OSX v10.12.2
Node v6.9.2

**Comment by ericsoco** [comment]
Ok, got the electron build script working. Not clear on how Browserify and Electron are working together, but I waded into the weeds and found that the `npm` `electron` package loads electron as a binary by following the relative path listed in `node_modules/electron/path.txt`. However, the path resolves to the build folder (instead of `node_modules/electron`). So, I added a `path.txt` to `/static` that resolves to the correct location.

Then, I had to figure out how to target the `electron` namespace from within an electron app bundled with browserify. Browserify overwrites `require` within the scope of the running application, but `window.require` still does the right thing, so I can use it to get at `electron` and from there to `webFrame.registerURLSchemeAsPrivileged('file')` ([see here](https://github.com/electron/electron/issues/3922) for more info).

I also set up these build scripts to add the browserify flags for electron [recommended here](http://maxgfeller.com/blog/2016/08/30/electron-browserify/), but they turned out to not be necessary; recording here for posterity:

```
    "dist-electron": "rm -rf ./dist-electron && mkdir dist-electron && npm run js:electron && npm run sass:electron && npm run static:electron",
    "js:electron": "browserify --ignore-missing --no-builtins --no-commondir --insert-global-vars=\"global\" --no-browser-field js/main.js -o dist-electron/bundle.js",
    "sass:electron": "node-sass --include-path scss scss/main.scss dist-electron/main.css",
    "static:electron": "cp -R static/ dist-electron/",
```

Next up is packaging it and seeing if it runs as a standalone.
